src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -68,7 +81,7 @@ public async Task<IActionResult> SubmitAnswer(SubmitAnswerDto submitAnswerDto, [
         else return RedirectToAction("GetQuestionById", new { id = nextQuestionId, submissionId = submissionId });
     }
 
-    private static Params? _ParseParameters(string parameters)
+    private static Params? _ParseParameters(string? parameters)
```

Comment:
Methods should not have _ for name

Reply:
Don't normally prefix methods with an underscore in C# mate, normally only private properties would have the prefix.